.github/workflows/build.yaml

```diff
@@ -0,0 +1,29 @@
+name: Build CLI
+
+on:
+  - pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Use Node.js LTS
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --non-interactive
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Build packages
+        run: yarn run turbo run build
```

Comment:
Note: We should define a top-level `build` script

Reply:
👍 yep good shout. Will do this in a separate RP